### PR TITLE
Add the PID to the log output

### PR DESCRIPTION
**Pull request by coolo**
Without it the prefork workers pollute each other and it makes hard to follow
what's going on

### CODE REVIEW

**Comment by aaannz** [comment]
Cool. :+1: 
I think I somewhere had a branch trying to do similar thing but using `on` callback instead. I was trying to add connection number there too so one could grep out log per request. Can't find it now though.

**Comment by coolo** [comment]
wow, we have a test case asserting the log output! :)
